### PR TITLE
Correct minor typos in tournament logistics doc

### PR DESCRIPTION
**Pull request by thbsp**
I noticed a few typos when reading over this myself so I ran the whole thing through a grammar 
checker and made a few obvious corrections.

### CODE REVIEW

File: docs/guide/tournament-logistics.rst [hunk 1]
```diff
@@ -463,7 +463,7 @@ Mistakes are made surprisingly often during results announcements. Again, this i
 - Have a single source for what is being read out — i.e. a printed list (recommended) or the tab site itself — but don't mix and match. If making slides (often a good idea for large/crowded venues) copy the data from the canonical source being announced.
 - Double check what is being read out against the tab site, and/or whatever draft lists were used to determine the adjudicator's break. Verify with the adjudication core that everyone who should be there is, and that nobody is missing.
 - Clarify what information should be on the print-outs and the general order in which things are read. For example, it might be easy to omit breaking adjudicator's institutions, to use ambiguous abbreviations over full institution names, or to have an inconsistent approach to how the information is read (i.e. whether it is read as *wins* then *team points* then *team name*).
-- Without revealing any details try and get at least some guidance on how to pronounce names that people are not familiar with pronounce.
+- Without revealing any details try and get at least some guidance on how to pronounce names that people are not familiar with.
```
Comment (tienne-B) [inline docs/guide/tournament-logistics.rst:466]: How about "Without revealing any details, try to get at least some guidance on how to pronounce unfamiliar names."?

File: docs/guide/tournament-logistics.rst [hunk 2]
```diff
@@ -318,16 +318,16 @@ If at all feasible you want to train that volunteers acting as runners and/or da
...
-  - Outline in general what happens during ballot collecting: when to do it, how to approach chairs, what do to if they are slow or delaying. You should raise the chance of chairs being belligerent and outline how they (and you) should deal with this.
-  - If you are having runners pre-check ballots it's a good idea to fill out a few 'bad' ballots to demonstrate the kinds of checking required. If you are using any communication systems (i.e. having runners mark of buildings as 'done' in an online system) go through that now also.
+  - It is critical that, as a group, you actually go to every one of the venue groups and identify all of the venue rooms that are listed so that everyone knows exactly where to go. This may take some time. But it is a good chance to both check those rooms actually exist and pre-identify any problems that might occur with runners and debaters finding them.
```
Comment (tienne-B) [inline docs/guide/tournament-logistics.rst:321]: Two options:
* "[...] you actually go to everyone of the venue [...]"
* "[...] you actually go to every venue [...]"

Splitting "everyone" isn't natural.

File: docs/guide/tournament-logistics.rst [hunk 3]
```diff
@@ -111,8 +111,8 @@ Conflicts/Clashes (registration/equity/adjudication core)
...
+- Conflicts should, ideally, *only be collected after a participants list has been published* and requests for people to nominate conflicts should also be sent out as few times as possible. Most people will only fill this form in once, so it is vital that when asked to nominate conflicts they have as much information as they need to do so comprehensively. Without a public and reasonably-complete participants list people will either nominate conflicts that are not present (wasting your time in cross-referencing data) or not realise someone is present and raise the conflict at a later, less opportune time.
```
Comment (tienne-B) [inline docs/guide/tournament-logistics.rst:115]: Neutral on this change; think "latter" can be used in relation to time (as used in the _Book of Mormon_ musical at least)

**Review by tienne-B — CHANGES_REQUESTED** [review]
Nice patch! Made me realize I'd never actually fully read that page! I've added a few comments that you could squash in.

As for the commit message, I would de-emphasize the "secret round" to "silent round", as that is also just a typo. How I make my commit messages is influenced by https://www.mediawiki.org/wiki/Gerrit/Commit_message_guidelines (note the suggested max of 80 characters for the title and 100 characters for each line in the message). This commit could be amended for these comments/suggestions.

File: docs/guide/tournament-logistics.rst (hunk 2 above)
Comment (czlee) [reply docs/guide/tournament-logistics.rst:321]: "every one of", as a synonym for "each of", is idiomatic ([Grammarly](https://www.grammarly.com/blog/everyone-vs-every-one/), [Writing Explained](https://writingexplained.org/everyone-vs-every-one-difference), [Lexico/Oxford](https://www.lexico.com/definition/every_one), [Collins](https://www.collinsdictionary.com/dictionary/english/every-one))

File: docs/guide/tournament-logistics.rst (hunk 3 above)
Comment (czlee) [reply docs/guide/tournament-logistics.rst:115]: I think it's a typo and should be corrected to "later" 🙂

When used to refer to time, it has a connation of "as opposed to the earlier part of a period", _e.g._ "the latter part of the 20th century" or I guess "the latter part of the tournament", which doesn't apply in this context.

**Comment by czlee** [comment]
Just confirming that "secret round" is a typo and should be "silent round", thanks guys 🙂

**Comment by czlee** [comment]
Also, sharp eyes @thbsp, thanks for putting this PR in!

File: docs/guide/tournament-logistics.rst (hunk 1 above)
Comment (thbsp) [reply docs/guide/tournament-logistics.rst]: Rephrased to "`- Without revealing any details, try to get some guidance on how to pronounce unfamiliar names.`"

**Comment by thbsp** [comment]
I edited the commit titles and messages as suggested.
(What do I do with the requested changes? I do personally 'resolve conversation' or leave that to one of the core members?)

**Review by tienne-B — APPROVED** [review]

**Comment by tienne-B** [comment]
Looks good! You may click on Resolve when you feel that the discussion is over (but I've done it already). I'll squash the update commit and reflow the message before merging though.

**Pull request merged**